`AppSideNav`: fix scrolling issue and being able to focus hidden links issue (#2869)
hashicorp/design-system: .changeset/honest-experts-battle.md

---
"@hashicorp/design-system-components": patch
---

> `AppSideNav` - fixed bug where scrolling was blocked when the `AppSideNav` was expanded on desktop views. Also fixed bug where able to focus links that are visually hidden.

Reviewer: ```suggestion
`AppSideNav` - Fixed bug where scrolling was blocked when the `AppSideNav` was expanded on desktop views. Also fixed bug which allowed user to focus links that were visually hidden.
```

Reviewer: Some of the wording above seemed a little awkward so I edited to try to clarify. (Feel free to tweak further.)

Author: This is definitely better!
